molecule/provisioner/ansible/playbooks/vagrant/prepare.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Prepare
+  hosts: all
+  become: true
+  gather_facts: false
+  tasks:
+    - name: Install some basic packages because Vagrant machines are bare
+      raw: |
+        if [ -x "$(command -v apt-get)" ]
+        then
+          apt-get update && apt-get install -y ca-certificates curl python sudo
```

Comment:
This (and the similar tasks below) installs different packages that are not required by Ansible by default compared to before with very little insight for the user by default which ones are actually being installed.

I'd rather see the LXD provider become more explicit than moving the `prepare` step to something internal to molecule...